File: Java/fibonacci.java
import java.util.*;

class Fibonacci { 

    static int fib(int n) 
    {
        if (n <= 1) 
            return n;
        return fib(n-1) + fib(n-2);
    } 

    public static void main(String args[]) 
    { 
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter value of N: ")
        int n = sc.nextInt();
        fib(n);
    } 
}

File: Python/fibonacci.py
#fibonacci series
def fibonacci(n):
    default=[0,1]
    for i in range(0,n-1):
        default.append(default[i]+default[i+1])
    print(default)

if __name__=="__main__":
    print("Enter the value of n")
    n=int(input())
    fibonacci(n)
